npm-scripts.js

```diff
@@ -190,3 +210,22 @@ function execute(command)
 		process.exit(1);
 	}
 }
+
+function installMsysMake()
+{
+	let res = spawnSync('where', [ 'python3.exe' ]);
+
+	if (res.status !== 0)
+	{
+		res = spawnSync('where', [ 'python.exe' ]);
+		if (res.status !== 0)
+		{
+			// eslint-disable-next-line no-console
+			console.error('Cannot find Python executable');
```

Comment:
Please honor the syntax of other console calls in this file.

```suggestion
			console.error('`npm-scripts.js [ERROR] cannot find Python executable');
```